stdlib/2/multiprocessing/__init__.pyi

```diff
@@ -1,11 +1,29 @@
+from typing import Optional
+
 from multiprocessing.process import Process as Process, current_process as current_process, active_children as active_children
 from multiprocessing.util import SUBDEBUG as SUBDEBUG, SUBWARNING as SUBWARNING
+import Queue as queue
```

Comment:
I think this should be underscored (either `_queue` or `_Queue` as you wish). Otherwise type checkers will treat  `queue` as externally visible (i.e. imported for re-export). You can read about special nature of `... as ...` imports in stubs in PEP 484, if interested.